tools/flaky-test-retryer/handler.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"log"
+
+	// TODO: remove this import once "k8s.io/test-infra" import problems are fixed
+	// https://github.com/test-infra/test-infra/issues/912
+	"github.com/knative/test-infra/tools/monitoring/prowapi"
+)
+
+// TODO(TrevorFarrelly): supported repos depend on flaky-test-reporter's
+// configuration. Use the reporter's config.yaml to build this list.
+var repos = []string{"serving"}
+
+// validStatus checks that the message signals a failure
+func validStatus(msg *prowapi.ReportMessage) bool {
```

Comment:
Suggest passing `status` straight since this is what this function is called
nit: `validStatus` is a bit misleading to me. sounds like `wantedStatus` or `expectedStatus`.